fuzz/src/chanmon_consistency.rs

```diff
@@ -776,8 +776,26 @@ pub fn do_test(data: &[u8]) {
 				nodes[2] = node_c.clone();
 				monitor_c = new_monitor_c;
 			},
-			0x22 => send_payment_with_secret!(nodes[0], (&nodes[1], chan_a), (&nodes[2], chan_b)),
-			0x23 => send_payment_with_secret!(nodes[2], (&nodes[1], chan_b), (&nodes[0], chan_a)),
+			0x22 => send_payment!(nodes[0], (&nodes[1], chan_a), 10),
+			0x23 => send_payment!(nodes[1], (&nodes[0], chan_a), 10),
+			0x25 => send_payment!(nodes[1], (&nodes[2], chan_b), 10),
+			0x26 => send_payment!(nodes[2], (&nodes[1], chan_b), 10),
+			0x27 => send_payment!(nodes[0], (&nodes[1], chan_a), (&nodes[2], chan_b), 10),
+			0x28 => send_payment!(nodes[2], (&nodes[1], chan_b), (&nodes[0], chan_a), 10),
+			0x29 => send_payment!(nodes[0], (&nodes[1], chan_a), 1_000),
+			0x2a => send_payment!(nodes[1], (&nodes[0], chan_a), 1_000),
+			0x2b => send_payment!(nodes[1], (&nodes[2], chan_b), 1_000),
+			0x2c => send_payment!(nodes[2], (&nodes[1], chan_b), 1_000),
+			0x2d => send_payment!(nodes[0], (&nodes[1], chan_a), (&nodes[2], chan_b), 1_000),
+			0x2e => send_payment!(nodes[2], (&nodes[1], chan_b), (&nodes[0], chan_a), 1_000),
+			0x2f => send_payment!(nodes[0], (&nodes[1], chan_a), 100_000),
+			0x30 => send_payment!(nodes[1], (&nodes[0], chan_a), 100_000),
+			0x31 => send_payment!(nodes[1], (&nodes[2], chan_b), 100_000),
+			0x32 => send_payment!(nodes[2], (&nodes[1], chan_b), 100_000),
+			0x33 => send_payment!(nodes[0], (&nodes[1], chan_a), (&nodes[2], chan_b), 100_000),
+			0x34 => send_payment!(nodes[2], (&nodes[1], chan_b), (&nodes[0], chan_a), 100_000),
+			0x35 => send_payment_with_secret!(nodes[0], (&nodes[1], chan_a), (&nodes[2], chan_b)),
```

Comment:
I haven't materially started fuzzing send_payment_with_secret yet, so it doesn't matter here, but in the future, generally best to avoid changing the fuzz format as it invalidates(ish) all current test cases/corpus. In full_stack_target we even have a super obnoxious test case that tests that a manually-generated fuzz input hits a bunch of the code.

Reply:
good to know!